Build.PL

```diff
@@ -24,6 +24,7 @@ my $builder = Module::Build->new(
         'JSON::XS'           => 0,
         'MIME::Lite'         => 0,
         'Net::SMTP::SSL'     => 0,
+        'Text::CSV'          => 0,

```

Comment:
I wonder if we can start plugging modules conditionally to minimize the number of dependencies that people need to just install and run Serge. This is not CSV plugin-specific, but just a general thought I have, seeing that we increase the number of . People will probably need only a few parser plugins, but now they have to download all parser modules at once. I'm already [doing this with Carp::Always](https://github.com/evernote/serge/blob/217264a57565b0ce0d50cc48bc6b5566e0b45a0f/lib/Serge/Application.pm#L49-L50) and it feels like the approach I would like to see more often if other places as well.

This is how it might work:

1. The module is only required for testing (dependency moved into `build_requires`)
2. In `validate_data` function, say:
```
eval('use Text::CSV;');
die "ERROR: To use parse_csv parser, please install Text::CSV module (run 'cpan Text::CSV')\n" if $@; 
```

This will catch the missing modules early, at the config validation time, and give users a somewhat friendly message

3. In `parse` function, say:

```
eval('use Text::CSV;');
die $@ if $@;
```

This will simply die with the original error message.